third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -2576,6 +2582,7 @@ func flattenPrivateClusterConfig(c *containerBeta.PrivateClusterConfig) []map[st
 			"enable_private_endpoint": c.EnablePrivateEndpoint,
 			"enable_private_nodes":    c.EnablePrivateNodes,
 			"master_ipv4_cidr_block":  c.MasterIpv4CidrBlock,
+			"peering_name":            c.PeeringName,
```

Comment:
TPG uses `containerBeta` so PeeringName should be an empty string (or scarily possibly a non-empty string). I think Terraform will throw an error when trying to set a field that isn't in the schema here.

Reply:
Thanks, think I fixed it - I'll run the container cluster tests in both GA and beta just in case.